### DISCUSSION TITLE
Where is the  front end code of huly?

### DISCUSSION THREAD

**Discussion by yadhukrishnasm** [hcengineering/platform#7093] [General]
I would like to make some changes to the ui and couldn't find the file , if any one can help me with anything. I would be delighted, if what I asked is an idiotic  just let me know, ok

**Comment by vasani-arpit** [comment]
#### Updated answer
https://github.com/hcengineering/platform/discussions/7093#discussioncomment-11238228

#### old answer
~~I think it is in https://github.com/hcengineering/platform/tree/develop/packages/ui~~
although I haven't personally run to confirm.

**Reply by BykhovDenis** [reply]
No. plugins/*-resources and packages/ui packages/presentation
